### ISSUE TITLE
rnni method of tree class throws error

### ISSUE THREAD

**Issue by eik-dahms** [kgori/treeCl#15]
Hi,
I am trying to use the random nni method of the tree class and I get the following error:
(I am using the current master branch version )

```
File "PATH/treeCl/tree.py", line 1146, in rnni 
nni.rnni(**kwargs)
File "PATH/treeCl/tree.py", line 344, in rnni
n = self.choose_node(weighted_choice, transform)
File "PATH/treeCl/tree.py", line 302, in choose_node
return weighted_choice(list(zip(self.valid_nodes, weights))
TypeError: 'bool' object is not callable

```
As one can see it tries to make a call to the boolean variable weighted_choice. It's seems there was used a wrong function name?

This code snippet should reproduce the error

```
from treeCl import tree as tr

in_tree = tr.Tree(newick="((((T4:42.9474018906,T10:42.9474018906):112.903906732,(T6:14.3433500048,(T2:1.53929863217,T5:1.53929863217):12.8040513726):141.507958618):22.1730692315,T9:178.024377854):34.9689886128,(T3:190.011180702,((T1:0,T8:0):147.182729024,T7:147.182729024):42.8284516785):22.9821857647):2.44503258424;")
nni_tree = in_tree.rnni(times=1)
```

**Comment by kgori** [comment]
Hi,
I confirm that I also get this error. I'll have a fix up in a few minutes.

**Comment by eik-dahms** [comment]
Thanks! 

And btw - the same thing seems to be happen in the ILS class - line 443

**Comment by kgori** [comment]
Thanks for the update. I think the ILS and NNI errors have the same source, so both should be fixed.

**Comment by kgori** [comment]
The fix should be on PyPI now (treeCl version 0.1.31). I'll leave this issue open for another day in case you still have any problems.

**Comment by eik-dahms** [comment]
Thanks - it is working now.
